fairlearn/postprocessing/_threshold_optimizer.py

```diff
@@ -60,52 +74,58 @@
 # These metrics are also required to be "moments" in the same sense as
 # required by fairlearn.reductions, so that the interpolation is possible.
 SIMPLE_CONSTRAINTS = {
-    'selection_rate_parity': 'selection_rate',
-    'demographic_parity': 'selection_rate',
-    'false_positive_rate_parity': 'false_positive_rate',
-    'false_negative_rate_parity': 'false_negative_rate',
-    'true_positive_rate_parity': 'true_positive_rate',
-    'true_negative_rate_parity': 'true_negative_rate',
+    "selection_rate_parity": "selection_rate",
```

Comment:
Another one out of curiosity: is double quote preferred for some reason? I had never heard of any guidance.

Reply:
Yep, `black` makes things double quotes.